exporters/pelorus/deserialization/__init__.py

```diff
@@ -223,9 +217,6 @@ def _extract_optional_type(type_: type) -> Optional[type]:
     """
     If this type annotation is Optional[T], then returns the type T.
     Otherwise, returns None.
-
-    >>> assert _extract_optional_type(Optional[int]) == int
-    >>> assert _extract_optional_type(int) is None
```

Comment:
I don't think we should remove the doctests. They're just as much documentation as they are tests.

What we can look into later is pytest's doctest support: https://docs.pytest.org/en/7.1.x/how-to/doctest.html

Reply:
I look into it

Reply:
added it :)